src/components/Form/Form.js

```diff
@@ -35,10 +35,34 @@ const Form = (props) => {
       selfDescription: selfDescription,
       feeling: feeling
     }
-    setPrescriptionType(determinePrescriptionType(inputs));
+    const type = determinePrescriptionType(inputs);
+    setPrescriptionType(type);
     console.log('TYPE', prescriptionType);
   }
 
+  const selfDescriptorOptions = [
+    { label: "procrastinator", value: "procrastinator" },
+    { label: "perfectionist", value: "perfectionist" },
+    { label: "adventurous", value: "adventurous"},
+    { label: "over-cautious", value: "over-cautious" },
+    { label: "sloppy", value: "sloppy" },
+    { label: "meticulous", value: "meticulous" },
+    { label: "self-concious", value: "self-concious" },
+    { label: "confident", value: "confident" },
+    { label: "anxious", value: "anxious" },
+    { label: "lazy", value: "lazy" },
+    { label: "obsessive", value: "obsessive" },
+    { label: "imaginative", value: "imaginative"},
+    { label: "hard-working", value: "hard-working" },
+    { label: "boring", value: "boring" },
+    { label: "brilliant", value: "brilliant" },
+    { label: "fast-learner", value: "fast-learner" },
+    { label: "slow-learner", value: "slow-learner" },
+    { label: "analytical", value: "analytical" },
+    { label: "insightful", value: "insightful" },
+    { label: "creative", value: "creative" }
+  ]
```

Comment:
could set up more of the form content like this -- would it make this component more readable. It's pretty MASSIVE at the moment